src/shaders/terrain_raster.vertex.glsl

```diff
@@ -24,8 +23,7 @@ void main() {
     vec2 decodedPos = a_pos - vec2(skirt * skirtOffset, 0.0);
     gl_Position = u_matrix * vec4(decodedPos, elevation, 1.0);
 
-#ifdef FOG
-    vec4 depthPos = u_cam_matrix * vec4(decodedPos, elevation, 1.0);
-    v_depth = length(depthPos.xyz);
+#if defined( FOG ) && !defined( RENDER_TO_TEXTURE )
```

Comment:
I don't think this code path can have RENDER_TO_TEXTURE

Reply:
Ah, good catch. Thanks for the detailed review!!